daemon/config/config_linux.go

```diff
@@ -87,9 +87,7 @@ type Config struct {
 	NoNewPrivileges      bool                      `json:"no-new-privileges,omitempty"`
 	IpcMode              string                    `json:"default-ipc-mode,omitempty"`
 	CgroupNamespaceMode  string                    `json:"default-cgroupns-mode,omitempty"`
-	// ResolvConf is the path to the configuration of the host resolver
-	ResolvConf string `json:"resolv-conf,omitempty"`
```

Comment:
Sorry - I missed this, or forgot to reply.

It's quite a while since I looked at this PR, so had to remind myself ... before this change, if the host's resolver was systemd's `127.0.0.53` we couldn't point at it from a container's `resolv.conf` on the default network. So, in that case, we based our config on systemd's alternative `resolv.conf`.

Now, even on the default network the internal resolver can use the systemd resolver directly, we don't need to know about its upstreams - and we only ever read `/etc/resolv.conf` (ignoring the undocumented env-var that's intended for tests to use).

So, I don't think continuing to log it would be very useful.